lib/spark.js

```diff
@@ -135,7 +135,7 @@ export default class Spark {
         this.#activity.onError()
         console.error(err)
       }
-      await sleep(1_000)
+      await sleep(10_000)
```

Comment:
Should we extract this into a constant defined at the top of this file to make it easier to find?

Reply:
good idea!